### ISSUE TITLE
Error processing functions created with Rcpp

### ISSUE THREAD

**Issue by wlandau** [duncantl/CodeDepends#31]
I think this is responsible for https://github.com/ropensci/drake/issues/344.

```r
library(Rcpp)
sourceCpp(
  code = '
  #include <RcppArmadillo.h>
  // [[Rcpp::depends(RcppArmadillo)]]
  // [[Rcpp::export]]
  arma::vec Mahalanobis(arma::mat x, arma::rowvec center, arma::mat cov){
  int n = x.n_rows;
  arma::mat x_cen;
  x_cen.copy_size(x);
  for (int i=0; i < n; i++) {
  x_cen.row(i) = x.row(i) - center;
  }
  return sum((x_cen * cov.i()) % x_cen, 1);
  }'
)
Mahalanobis(cbind(1, 2:3), 3:4, diag(2))
#>      [,1]
#> [1,]    8
#> [2,]    5
CodeDepends::getInputs(Mahalanobis)
#> Warning: replacing previous import 'graph::addNode' by 'XML::addNode' when
#> loading 'CodeDepends'
#> Warning: replacing previous import 'graph::plot' by 'graphics::plot' when
#> loading 'CodeDepends'
#> Error in .local(e, collector, basedir, reset, formulaInputs, ...): don't know about NativeSymbol
```

**Comment by gmbecker** [comment]
This should be fixed in 0.5.5.5 Please reopen if you run into any further problems.

**Comment by wlandau** [comment]
Awesome, thanks Gabe!
